mibidata/combine_tiffs.py

```diff
@@ -0,0 +1,204 @@
+        out: Optionally, a path to a location for saving the combined TIFF. If
+           not specified, defaults to 'combined.tiff' inside the input folder.
+        run_label: Optionally, a custom run label for the `run` property of the
+            image.
+    """
+    panel_df = panels.read_csv(panel_path)
+    panel_name, _ = os.path.splitext(os.path.basename(panel_path))
+    tiff_files = os.listdir(input_folder)
+
+    fovs, calibration = runs.parse_xml(run_path)
+    point_number = int(point[5:])
+    try:
+        fov = fovs[point_number]
+    except IndexError:
+        raise IndexError('Point{} not found in run xml.'.format(point_number))
+    if fov['date']:
+        run_date = datetime.datetime.strptime(
+            fov['date'], '%Y-%m-%dT%H:%M:%S').date()
+    else:
+        run_date = datetime.datetime.now().date()
+
+    image_data = []
+    for i in panel_df.index:
+        tiff_path = os.path.join(
+            input_folder, _match_target_filename(tiff_files,
+                                                 panel_df['Target'][i]))
+        data = _load_single_channel(tiff_path)
+        image_data.append(data)
+
+    image_data = np.stack(image_data, axis=2)
+
+    image = mi.MibiImage(image_data,
+                         list(zip(panel_df['Mass'], panel_df['Target'])))
+
+    image.size = int(size)
+    image.coordinates = (fov['coordinates'])
+    image.filename = fov['run']
+    image.run = run_label if run_label else fov['run']
+    image.version = tiff.SOFTWARE_VERSION
+    image.instrument = instrument
+    image.slide = slide
+    image.dwell = fov['dwell']
+    image.scans = fov['scans']
+    image.aperture = aperture
+    image.point_name = fov['point_name']
+    image.folder = fov['folder']
+    image.tissue = tissue
+    image.panel = panel_name
+    image.date = run_date
+    image.mass_offset = calibration['MassOffset']
+    image.mass_gain = calibration['MassGain']
+    image.time_resolution = calibration['TimeResolution']
+
+    if out is None:
+        out = os.path.join(input_folder, 'combined.tiff')
+
+    tiff.write(out, image, multichannel=True)
+
+
+if __name__ == '__main__':
+
+    description = ('Generates a single multiplexed MIBItiff from a folder '
+                   'containing individual single-channel TIFFs without MIBI '
+                   'metadata. Note that if the single-channel TIFFs already '
+                   'have the MIBItiff metadata, then a much simpler way to '
+                   'merge them is to use combine_tiffs.merge_mibitiffs '
+                   'instead.')
+
+    parser = argparse.ArgumentParser(description=description)
+    parser.add_argument(
+        'folder',
+        help='Folder containing single-channel TIFF files with a .tif or '
+             '.tiff extension.',
+    )
+    parser.add_argument(
+        'run_xml', help='Path to a run XML file.'
+    )
+    parser.add_argument(
+        'point', help='The point number in the run, such as Point1 or Point2.'
```

Comment:
A potential pain point for existing users (as it was for me earlier today) was that I was unsure what to use here, so I opened the `sample_parameters.csv` file we packaged with the original version (as I did for other required/optional args). That specified the `point_name` which was `1245_Bottom_R1C4_Tonsil` in that example, which of course fails here because an int is parsed to index the FOV. Not saying this needs to be changed here, but could be something that comes up.

Reply:
Good point (pun intended). Is there a change to the help text you'd suggest?

Reply:
How about something like "... such as Point1 or Point2. This should match the name of the folder created for the raw data."

Reply:
Works for me. That should be clear enough for existing users.

Reply:
Cool, I did that plus updated the error message. You approval is requested again.